### PR TITLE
Split related data page into separate pages

### PR DESCRIPTION
**Pull request by roji**
Our related data page has grown considerably, and gotten to the point where it seems better to split it into different sub-pages (eager, explicit, lazy) so as to avoid overwhelming the user. This also allows each loading strategy to have its own in-page navigation links, etc.

Also moved related data up in the table of contents, as it seems like on of the first thing people need to know about.

### CODE REVIEW

File: entity-framework/core/querying/related-data/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Loading Related Data - EF Core
+description: Different strategies for loading related data with Entity Framework Core
+author: rowanmiller
+ms.date: 10/27/2016
```
Comment (AndriySvyryd) [inline entity-framework/core/querying/related-data/index.md:5]: Update author and date

**Review by AndriySvyryd — APPROVED** [review]

File: entity-framework/core/querying/related-data/eager.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+You can combine all of the calls to include related data from multiple levels and multiple roots in the same query.
+
+[!code-csharp[Main](../../../../samples/core/Querying/RelatedData/Sample.cs#IncludeTree)]
+
+You may want to include multiple related entities for one of the entities that is being included. For example, when querying `Blogs`, you include `Posts` and then want to include both the `Author` and `Tags` of the `Posts`. To include both, you need to specify each include path starting at the root. For example, `Blog -> Posts -> Author` and `Blog -> Posts -> Tags`. It doesn't mean you'll get redundant joins; in most cases, EF will combine the joins when generating SQL.
+
+[!code-csharp[Main](../../../../samples/core/Querying/RelatedData/Sample.cs#MultipleLeafIncludes)]
+
+## Single and split queries
```
Comment (smitpatel) [inline entity-framework/core/querying/related-data/eager.md:41]: Need to update the fwlink to point here https://go.microsoft.com/fwlink/?linkid=2134277

Comment (roji) [reply entity-framework/core/querying/related-data/eager.md:41]: /cc @JeremyLikness 

Do we have a list somewhere tracking needed changes or something?

Comment (smitpatel) [reply entity-framework/core/querying/related-data/eager.md:41]: I am doing exception message review. 😆

Comment (JeremyLikness) [reply entity-framework/core/querying/related-data/eager.md:41]: @roji I don't have a master list of fwlinks to update, only what people reach out to me to ask.

Comment (smitpatel) [reply entity-framework/core/querying/related-data/eager.md:41]: @roji @JeremyLikness - Can we get this link target updated? It has been pushed to live. And exception message leaves in source code already.

Comment (roji) [reply entity-framework/core/querying/related-data/eager.md:41]: This is just about changing the fwlink right?

Comment (smitpatel) [reply entity-framework/core/querying/related-data/eager.md:41]: Yes.

Comment (JeremyLikness) [reply entity-framework/core/querying/related-data/eager.md:41]: @roji @smitpatel what is the URL it needs to point to?

Comment (smitpatel) [reply entity-framework/core/querying/related-data/eager.md:41]: https://docs.microsoft.com/ef/core/querying/related-data/eager#single-and-split-queries

Comment (JeremyLikness) [reply entity-framework/core/querying/related-data/eager.md:41]: Thanks. This is done.

**Pull request merged**